.travis.yml

```diff
@@ -24,6 +24,7 @@ install:
   - python --version
   - pip install -r ci/pip_requirements.txt
   - pip install . --no-deps
+  - pip install hdmedians
```

Comment:
not big deal, but could you put `hdmedians` into `ci/pip_requirement.txt` and remove this line?